rpc/v1/openapi.yaml

```diff
@@ -91,6 +91,122 @@ paths:
           content:
             text/plain:
               example: "description of failure"
+  /v1/servicer/relay:
```

Comment:
Make sure to rebase on top of @h5law's recent changes since I anticipate merge conflicts.